nifi-nar-bundles/nifi-redis-bundle/nifi-redis-extensions/pom.xml

```
@@ -82,6 +82,11 @@
             <artifactId>nifi-record-path</artifactId>
             <version>2.0.0-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-ssl-context-service-api</artifactId>
+            <scope>compile</scope>
```

This dependency should be marked as provided.
```suggestion
            <scope>provided</scope>
```

---

Sure, I just moved this declaration up, as it was listed under the "test dependencies" comment.

---

Thanks!